### PR TITLE
memory usage excludes buffer and cache.

### PR DESCRIPTION
**Pull request by 2vg**
ビルドしていないので要確認

### CODE REVIEW

**Comment by syuilo** [comment]
Thanks!

File: src/daemons/server-stats.ts [hunk 1]
```diff
@@ -1,5 +1,5 @@
 import * as os from 'os';
-const osUtils = require('os-utils');
+const sysUtils = require('systeminformation');
```
Comment (acid-chicken) [inline src/daemons/server-stats.ts:2]: I'm feeling the below code is better than that.
```js
import * as sysUtils from 'systeminformation';
```

File: src/daemons/server-stats.ts [hunk 2]
```diff
@@ -18,25 +18,60 @@ export default function() {
...
+// TOTAL MEMORY STAT
+async function totalMem() {
+  try {
+    const data = await sysUtils.mem();
+    return data.total;
+  }
+  catch(error) {
+    console.error(error);
+  }
+}
```
Comment (acid-chicken) [inline src/daemons/server-stats.ts:77]: Please add newline here.
https://github.com/syuilo/misskey/blob/933255179151ca0ee6ea7fcc20299e54b2c1d0a2/.editorconfig#L7

File: src/daemons/server-stats.ts (hunk 1 above)
Comment (AyaMorisawa) [reply src/daemons/server-stats.ts:2]: Done

File: src/daemons/server-stats.ts (hunk 2 above)
Comment (AyaMorisawa) [reply src/daemons/server-stats.ts:77]: Done

**Pull request merged**